README.md

```diff
@@ -132,3 +133,55 @@ Using this controller as *sync* has influence on all other controller, therefore
    ```
    ros2 launch workshop_bringup async_controllers.launch.xml
    ```
+
+
+
+# Task 4: Multi-robot scalable parameter handling
+
+This task demonstrates how to integrate multiple robots under different controller manager instances and how to handle parameters for better scalability of controller's configuration.
+
+#### Scenario
+Using PAL TIAGo robot we are going to repeat the same controller chaining as in deploying on a fleet of robots. The idea is to make the setup work on different robots working on different namespaced controller_manager. 
+
+#### What to focus on
+Make sure to understand the controller configuration present in the [chaining_controllers_tiago1.yaml](./workshop_bringup/config/chaining_controllers_tiago1.yaml) and [chaining_controllers_generic.yaml](./workshop_bringup/config/chaining_controllers_generic.yaml).
+
+#### Running the example
+
+1. In a terminal where your workspace is sourced start launch file:
+   ```
+   ros2 launch workshop_bringup tiago_chaining.launch.launch.xml namespace:=tiago1
+   ```
+
+   You should see that the different controllers are failing as the initial setup defined in the [chaining_controllers.yaml](./workshop_bringup/config/chaining_controllers.yaml) is not enough to work with the different controller managers setup
+
+2. In a terminal where your workspace is sourced start launch file:
+   ```
+   ros2 launch workshop_bringup tiago_chaining.launch.launch.xml namespace:=tiago1 config_file:=chaining_controllers_tiago1
+   ```
+
+   Now, launching the [chaining_controllers_tiago1.yaml](./workshop_bringup/config/chaining_controllers_tiago1.yaml) configuration file works because all the parameters are namespaced to the robot's serial number and it works. But!!!! you need to hardcode the namespacing in each and every robot's configuration file. If this works for you, then :thumbsup:. If not, the point 3 is for you!
```

Comment:
```suggestion
   Now, launching the [chaining_controllers_tiago1.yaml](./workshop_bringup/config/chaining_controllers_tiago1.yaml) configuration file works because all the parameters are namespaced to the robot's serial number. But !!! :point_up: !!!  you need to hardcode the namespacing in each robot configuration file. If this works for you, then :thumbsup:. If not, point 3 is for you!
```